bin/handler-opsgenie.rb

```diff
@@ -118,6 +120,19 @@ def create_alert
                      teams:       json_config['teams'])
   end
 
+  def event_priority
+    return nil unless json_config['priority']
```

Comment:
I'd suggest returning `false` rather or `DEFAULT_PRIORITY` returning `nil` is often harder to trace if it is a bug or intended.

Reply:
Excellent point.  PR updated.